config/vufind/LibGuidesAZ.ini

```diff
@@ -0,0 +1,16 @@
+; Settings for the LibGuides AZ Databases List are likely to be the same as those 
+; for the canonical "guides" part of LibGuides, defined in LibGuides.ini.
+; But override here if needed.
+[Parent_Config]
+relative_path = ./LibGuides.ini
+
+; ---------- config.ini settings ----------
```

Comment:
I don't think you need this comment; these settings don't come from config.ini. I'd either change this to say "LibGuides.ini" or remove it entirely. Given the Parent_Config settings right above, removing it is probably fine.

Reply:
Yeah typo, I meant Libguides.ini obviously.  But you're right it's redundant, I removed it.

Reply:
Thanks!